collectors/template/collector.js.template

```diff
@@ -0,0 +1,96 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2019, Alert Logic, Inc
+ * @doc
+ *
+ * {{type}} class.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+'use strict';
+
+const moment = require('moment');
+const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
+
+
+const typeIdPaths = [
+    // enter your type paths in the form { path: ['myKey'] }
+];
+
+const tsPaths = [
+    // enter your timestamp paths in the form { path: ['myKey'] }
+];
+
+
+class {{Type}}Collector extends PawsCollector {
+    constructor(context, creds) {
+        super(context, creds, '{{type}}');
+    }
+    
+    pawsInitCollectionState(event, callback) {
+        const startTs = process.env.paws_collection_start_ts ? 
+                process.env.paws_collection_start_ts :
+                    moment().toISOString();
+        const endTs = moment(startTs).add(this.pollInterval, 'seconds').toISOString();
+        const initialState = {
+            since: startTs,
+            until: endTs,
+            poll_interval_sec: 1
+        };
+        return callback(null, initialState, 1);
+    }
+    
+    pawsGetLogs(state, callback) {
+        let collector = this;
+        console.info(`{{TYPE}}000001 Collecting data from ${state.since} till ${state.until}`);
```

Comment:
I think we want to limit prefix to 4/5 chars.
same below
Maybe pass additional parameter to 
```
run `npm run create-collector <<name>> <<version>> <<log-prefix>>`
```

Reply:
@JDragovichAlertLogic  is `{{TYPE}}` limited to 4/5 chars?

Reply:
Yep. I've limited it in the validation part of the creation script.